resources/joplin.edn

```diff
@@ -1,9 +1,9 @@
-{:migrators    {:sql-mig    "resources/migrations"}
+{:migrators    {:sql-mig    "/migrations"}
 
  :seeds        {:sql-seed  "seeds.sql/run"}
  
- :databases    {:psql-dev  {:type :sql, :url "jdbc:postgresql:postgres?user=postgres&password=postgres"}
-                :psql-test {:type :sql, :url "jdbc:postgresql:postgres_test?user=postgres&password=postgres"}}
+ :databases    {:psql-dev  {:type :sql, :url #envf ["jdbc:postgresql:playground_dev?user=%s&password=%s" DEV_USER DEV_PASSWORD]}
```

Comment:
rename the `DEV_` prefix to `manandearth_playground_` (in caps or not - as you wish).

Generally always prefix env vars like that, it's very common that one ends up having dozens of them so it's good to know what's what